userdocs/src/gitops-quickstart.md

```diff
@@ -92,32 +83,32 @@ kube-system   kube-proxy-qzcmk           1/1     Running   0          45s
 
 The following command will set up your cluster with:
 
-- [Flux](https://github.com/fluxcd/flux),
```

Comment:
These are the latest changes AFAIK, this seems like a regression. This whole file needs to be updated with what's in master